hack/update-generated-protobuf.sh

```diff
@@ -37,7 +37,13 @@ function prereqs() {
   KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
   KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
   DOCKER_MOUNT_ARGS=(
-    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
+    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
```

Comment:
There is not a way to all-but-one.  But we should just make all the tooling
not run as root in the first place.  I tried it, it's a little more than
trivial.  First PR is in flight.  Not sure how much time I'll get...

On Tue, May 10, 2016 at 12:10 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In hack/update-generated-protobuf.sh
> <https://github.com/kubernetes/kubernetes/pull/25354#discussion_r62621882>
> :
>
> > @@ -37,7 +37,13 @@ function prereqs() {
> >    KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
> >    KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
> >    DOCKER_MOUNT_ARGS=(
> > -    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
> > +    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
>
> The latter.
>
> The problem is basically, that we want to avoid mounting the
> k8s.io/kubernetes/_output directory. If there is an easier way to mount
> all but this one, then it would be preferred. But I don't know how to do it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25354/files/9a8e4f69cd707441ea9f7996a983f9e30a30a5ce#r62621882>
>